PhpAmqpLib/Connection/AMQPSocketConnection.php

```diff
@@ -6,17 +6,19 @@
 class AMQPSocketConnection extends AbstractConnection
 {
     /**
-     * @param string $host
-     * @param int $port
-     * @param string $user
-     * @param string $password
-     * @param string $vhost
-     * @param bool $insist
-     * @param string $login_method
-     * @param null $login_response
-     * @param string $locale
-     * @param float $timeout
-     * @param bool $keepalive
+     * @param string    $host
+     * @param int       $port
```

Comment:
Don't know all the rules related to format for this repo but personally I prefer not to change the format of lines I don't touch. This way, the diff would have been cleaner and also the blame wouldn't by formatting stuff.
I suggest reverting the format to the original one, without aligning variable names.